src/main/java/net/conjur/api/clients/ResourceClient.java

```diff
@@ -20,28 +21,39 @@
 public class ResourceClient implements ResourceProvider {
 
     private WebTarget secrets;
-    private final Endpoints endpoints;
+    private Endpoints endpoints;
 
-    public ResourceClient(final Credentials credentials, final Endpoints endpoints) {
+    public ResourceClient(Credentials credentials, Endpoints endpoints) {
```

Comment:
Why the removal of `final` here?

Reply:
Sorry, this was removed by mistake. My IDE is normally configured to make all parameters, fields, and variables final, wherever possible. After I realized how many changes of that kind were made I wanted to remove that kind of change in code where it was the only change made.